modules/model-context-protocol-tools/con-understanding-model-context-protocol.adoc

```diff
@@ -0,0 +1,10 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="understanding-model-context-protocol"]
+= Understanding Model Context Protocol
+
+include::{docdir}/artifacts/snip-developer-preview.adoc[]
+
+The Model Context Protocol (MCP) server offers a standardized method for linking AI models and applications (MCP clients) with external systems. This connection facilitates access to information and workflows residing on those systems. MCP servers are responsible for defining the tools that AI applications can utilize to retrieve this data.
```

Comment:
I would leave out server from here, as the protocol is what offers the standardized method.
```suggestion
Model Context Protocol (MCP) offers a standardized method for linking AI models and applications (MCP clients) with external systems. This connection facilitates access to information and workflows residing on those systems. MCP servers are responsible for defining the tools that AI applications can utilize to retrieve this data.
```